src/platforms/java/common/performance/index.mdx

```diff
@@ -0,0 +1,103 @@
+
+<PlatformContent includePath="performance/enable-tracing" />
+
+If either of these options is set, tracing will be enabled in your app, and transactions will start getting captured automatically. (The two options are meant to be mutually exclusive, but if you do set both, <PlatformIdentifier name="traces-sampler" /> will take precedence.)
+
+As you're getting tracing set up, we recommend setting <PlatformIdentifier name="traces-sample-rate" /> to `1`, so all created transactions are sent to Sentry. Once you're done with testing, though, you'll probably want to consider either **lowering your <PlatformIdentifier name="traces-sample-rate" /> value, or switching to <PlatformIdentifier name="traces-sampler" />**, which will allow you to set the sample rate individually for each transaction. Without sampling, automatically-captured transactions can add up quickly. (The Spring Boot integration, for example, will send a transaction for every request made to the server.) Sampling allows you to send representative data without overwhelming either your system or your Sentry transaction quota.
+
+You can learn more about the <PlatformIdentifier name="traces-sample-rate" /> and <PlatformIdentifier name="traces-sampler" /> options in <PlatformLink to="/performance/sampling/">Sampling Transactions</PlatformLink>.
+
+## Manual Instrumentation
+
+To manually instrument certain regions of your code, you can create a transaction to capture them.
+
+The following example creates a transaction for a scope that contains an expensive operation (for example, `processItem`), and sends the result to Sentry:
+
+```java
+import io.sentry.Sentry;
+import io.sentry.SentryTransaction;
+import io.sentry.SpanStatus;
+
+SentryTransaction transaction = Sentry.startTransaction("transaction name");
+transaction.setOperation("task");
+try {
+    processItem();
+} catch (Exception e) {
+    transaction.setThrowable(e);
+    transaction.setSpanStatus(SpanStatus.INTERNAL_ERROR)
+} finally {
+    transaction.finish();
+}
+```
+
+### Adding More Spans to the Transaction
+
+The next example contains the implementation of the hypothetical `processItem` function called from the code snippet in the previous section. Our SDK can determine if there is currently an open transaction and add all newly created spans as child operations to that transaction. Keep in mind that each individual span also needs to be manually finished; otherwise, spans will not show up in the transaction.
+
+In cases where you want to attach Spans to an already ongoing Transaction you can use `Sentry#getSpan`. This method will return a `SentryTransaction` in case there is a running Transaction or a `Span` in case there is already a running Span, otherwise it returns `null`.
+
+You can choose the value of `operation` and `description`.
+
+```java
+import io.sentry.ISpan;
+import io.sentry.Span;
+import io.sentry.SpanStatus;
+
+void processItem(Item item):
+    ISpan span = Sentry.getSpan();
```

Comment:
This means the user would need to have bound the transaction to the scope, according to what I got from @HazAT so far. And we haven't documented that.

Do we need to show that on the previous example? Or if we're binding it automatically, I wonder how this works on Android: Not possible to have more than 1 transaction at the same time in the app?

Reply:
Transactions are bound to the scope https://github.com/getsentry/sentry-java/blob/main/sentry/src/main/java/io/sentry/Scope.java#L24

Reply:
@marandaneto maybe you can jump in and tell if/how this will work on Android

Reply:
tbh, I think it'd be a nice addition to be able to have multiple transactions at once for Mobile as well, but I don't see this as a huge blocker, at the end, it's a single user doing all the things (for most of the Apps), we can just document this limitation and tell people to go around with Spans if they need to measure different things (this is possible, right)?

In most of the cases, there will be a single transaction per screen anyway and if they want to create their own Hub for having multiple transactions, so they do.

The only concern I have right now is, a Hub is not ready for usage the way how it is, when creating a new Hub out of a fresh SentryOptions, a few things are not set like the Serializer, it's NoOp by default, it's only set on Sentry.init static class, what's about fixing this limitation, so at least we give a useful alternative for people?

Reply:
Regarding Hub initialization, since I don't know the background behind it I can't tell what are the side-effects, but perhaps @bruno-garcia can shine some light.

Reply:
well it's pretty simple, we've discussed this offline recently actually.

do Hub(SentryOptions(dsn)).captureEvent(SentryEvent()), nothing is sent due to missing default options on SentryOptions, it should work ootb, eg a Serializer is not set (NoOp) by default.